Packs/PAN-OS/Playbooks/PAN-OS_-_Job_-_Remove_Malicious_Domains_From_Sinkhole.yml

```diff
@@ -0,0 +1,508 @@
+  "19":
+    id: "19"
+    taskid: ed078f6d-529e-4243-8d35-a03182bdd5ad
+    type: condition
+    task:
+      id: ed078f6d-529e-4243-8d35-a03182bdd5ad
+      version: -1
+      name: Any domains have NOT been tagged recently?
+      description: Checks whether any of the checked domains was not tagged within the decay period. If domains have been recently tagged (E.g., after the date derived from the decay period input), they will not be untagged.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: indicators
+                filters:
+                - - operator: notIn
+                    left:
+                      value:
+                        simple: indicators.value
+                      iscontext: true
+                    right:
+                      value:
+                        simple: DomainsRecentlyTagged
+                      iscontext: true
+                accessor: value
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 900
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1345,
+        "width": 1360,
+        "x": -10,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: SinkholeTag
+  value:
+    simple: to_sinkhole
+  required: true
+  description: |-
+    The tag that will be removed from the tagged domain indicators.
+    This should be the tag that was used to export the domains to the EDL using the Generic Export Indicators Service integration, which is used by the firewall's sinkhole configuration.
+  playbookInputQuery:
+- key: DaysTaggedBeforeRemoval
+  value:
+    simple: "14"
+  required: true
+  description: |-
+    The "decay period" - how many days should pass since the domains were tagged, before removing the sinkhole tag from those domains.
+    The value should be a number of days. For example: 14.
+
+    Technical details: when specifying the decay period, the playbook will take into account multiple tagging/untagging activity in the domain indicator in XSOAR. This means that if you specified a value of "14" and a domain was tagged 15 days ago, but also untagged and then tagged again 10 days ago - it will not be untagged by the playbook, as tagging activity happened 10 days ago. It will, however, be untagged when the playbook runs 5 days afterwards, since at that time the domain will have reached the decay period again.
```

Comment:
```suggestion
    Technical details: When specifying the decay period, the playbook will take into account multiple tagging/untagging activities in the domain indicator in Cortex XSOAR. This means that if you specified a value of "14" and a domain was tagged 15 days ago, but also untagged and then tagged again 10 days ago - it will not be untagged by the playbook, as tagging activity happened 10 days ago. It will, however, be untagged when the playbook runs 5 days afterwards, since at that time the domain will have reached the decay period again.
```